bin/p2-rctl/main.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/square/p2/Godeps/_workspace/src/github.com/Sirupsen/logrus"
+	"github.com/square/p2/Godeps/_workspace/src/gopkg.in/alecthomas/kingpin.v1"
+
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/kp/rcstore"
+	"github.com/square/p2/pkg/labels"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/pods"
+	"github.com/square/p2/pkg/rc/fields"
+	"github.com/square/p2/pkg/util/net"
+	"github.com/square/p2/pkg/version"
+)
+
+const (
+	CMD_CREATE   = "create"
+	CMD_DELETE   = "delete"
+	CMD_REPLICAS = "set-replicas"
+	CMD_LIST     = "list"
+	CMD_GET      = "get"
+)
+
+var (
+	consulUrl   = kingpin.Flag("consul", "The hostname and port of a consul agent in the p2 cluster. Defaults to 0.0.0.0:8500.").String()
+	consulToken = kingpin.Flag("token", "The consul ACL token to use. Empty by default.").String()
+	headers     = kingpin.Flag("header", "An HTTP header to add to requests, in KEY=VALUE form. Can be specified multiple times.").StringMap()
+	https       = kingpin.Flag("https", "Use HTTPS").Bool()
```

Comment:
I was thinking about that, however nightmares of glog flags are convincing me that the answer is no. If kingpin had a way to make flags "silent" and then attach to the application, then I would consider it